express-static/index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const pubPath = path.join(__dirname, '/public');
+
+console.log(pubPath);
+
+app.use(express.static(pubPath));
+
+const port = 3000;
+
+app.listen(port, function (err) {
+  if (err) console.log('Error in server setup');
```

Comment:
This callback function won't be given an error, even if there is one so there is no need to check that here